src/neptune/metadata_containers/project.py

```diff
@@ -281,9 +287,16 @@ def fetch_runs_table(
             child_type=ContainerType.RUN,
             query=nql_query,
             columns=columns,
+            limit=limit,
         )
 
-    def fetch_models_table(self, *, columns: Optional[Iterable[str]] = None, trashed: Optional[bool] = False) -> Table:
+    def fetch_models_table(
+        self,
+        *,
+        columns: Optional[Iterable[str]] = None,
+        trashed: Optional[bool] = False,
+        limit: Optional[int] = None,
+    ) -> Table:
         """Retrieve models stored in the project.
 
         Args:
```

Comment:
@normandy7 I've just discovered that `Examples` under `columns` makes the whole docstrings unrecognizable in PyCharm. That's not for now but we should probably change that to something like `Sample usages`.